elliottlib/cli/__main__.py

```diff
@@ -96,8 +96,12 @@
 @click.option("--id", metavar='BUGID', default=[],
               multiple=True, required=True,
               help="Bugzilla IDs to add.")
+@click.option("--auto",
```

Comment:
i liked your `--all` idea better, but naming a var `all` could be bad... what about:
```suggestion
@click.option("--all", "remove_all",
```